### PR TITLE
Hit fission server for dataroot

### PR DESCRIPTION
**Pull request by dholms**
## Problem
DNS takes a while to propagate, and especially on initial lookups, it can take > 15 min

## Solution
Hit new server route for a user's DataRoot, if that fails, then go to DNS

### CODE REVIEW

File: src/data-root.ts [hunk 1]
```diff
@@ -25,6 +29,27 @@ export async function lookup(
...
+ * Get the CID of a user's data root from the Fission server.
+ *
+ * @param username The username of the user that we want to get the data root of.
+ */
+export async function lookupOnFisson(
+  username: string
+): Promise<CID | null> {
+  try {
+    const resp = await fetch(`${setup.endpoints.api}/user/data/${username}`)
+    const cid = await resp.json()
```
Comment (icidasset) [inline src/data-root.ts:42]: Does `resp.json()` work for plain-text responses? Or is `cid` an object?

**Review by icidasset — APPROVED** [review]
One question looks good otherwise

File: src/data-root.ts (hunk 1 above)
Comment (dholms) [reply src/data-root.ts:42]: yup works for plain-text responses :+1: 

We could do `resp.text()` but that adds quotes on either side so it'd have to be something like
`(await resp.text()).replace('"', '')`

Comment (dholms) [reply src/data-root.ts:42]: I just figured this was cleaner

Comment (icidasset) [reply src/data-root.ts:42]: Nice, didn't know that. Carry on 😄

Comment (dholms) [reply src/data-root.ts:42]: haha neither did I :smile: found out while coding this^

**Pull request merged**